src/app/features/to-do-list/to-do-list-canvas/to-do-list-canvas.component.ts

```diff
@@ -10,8 +10,8 @@ import {
 import { Observable } from 'rxjs';
 @Component({
   selector: 'app-canvas',
-  templateUrl: './canvas.component.html',
-  styleUrls: ['./canvas.component.scss'],
+  templateUrl: './to-do-list-canvas.component.html',
+  styleUrls: ['./to-do-list-canvas.component.scss'],
 })
 export class CanvasComponent implements AfterViewInit {
```

Comment:
Hai dimenticato di rinominare i nomi delle classi  in questo caso ToDoListCanvasComponent e dei selettori  app-to-do-list-canvas

Reply:
ho rinominato le calassi e i selettori